action/deployment/validate.go

```diff
@@ -28,12 +28,12 @@ func (c *Config) Validate() error {
 	if c.Action == "add" {
 		// check if deployment ref is set
 		if len(c.Ref) == 0 {
-			return fmt.Errorf("no deployment ref provided")
+			logrus.Debug("no ref provided. Using repo default branch...")
```

Comment:
Can we leave "deployment" in the message? Also, do you think we should make this a warning?

```suggestion
			logrus.Warn("no deployment ref provided. Using repo default branch...")
```